trunk/SUAVE/Analyses/Mission/Vary_Cruise/Given_State_of_Charge.py

```diff
@@ -0,0 +1,105 @@
+## @ingroup Analyses-Mission-Vary_Cruise
+# Given_Weight.py
+#
+# Created:  
+# Modified: Feb 2016, Andrew Wendorff
+
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Methods import Missions as Methods
+from SUAVE.Analyses.Mission import All_At_Once

```

Comment:
Here are some suggested names instead of "All_At_Once" 
-  All_At_Once_Mission 
- Full_Mission.

Reply:
These should stay. These are the two methods that are used to solve a mission. We mostly do Sequential_Segments (solve one segment at a time). However a variable mission uses All_At_Once (the mission is one segment).